Documentation/Installation/2_Installation.md

```diff
@@ -1,23 +1,39 @@
 # Extension Installation
 
-The installation boils down to three simple steps on both Local and Foreign:
-
-1. Copy the source code to typo3conf/ext/in2publish (Please see the in2code GitHub tutorial for more details).
-2. Activate the Extension via Extension Manager
-3. Setup IN2PUBLISH_CONTEXT for the current installation (see [Preparation](1_Preparation.md))
-4. Open the in2publish extension configuration (Click on "in2publish_core" in the Extension Manager) and set the path to your **Configuration.yaml** files (see next chapter for details).
-
-## New modules on local
+The installation requires the following steps on both the local and
+foreign system:
```

Comment:
"Local" and "Foreign" are the official names we use to identify the distinct system (see https://github.com/in2code-de/in2publish_core/blob/develop/Documentation/Introduction.md#environment)
These terms are names and must not be converted into adjectives (even if this has already been done incorrectly)

Reply:
Good point. Changed this just now.